Solutions/SlashNext SIEM/Data/Solution_SlashNextSIEM.json

```diff
@@ -0,0 +1,15 @@
+{
+    "Name" : "SlashNext SIEM",
+    "Author" : "SlashNext - support@slashnext.net",
+    "Logo" : "<img src=\"Playbook/logo/slashnext-logo.svg\" width=\"75px\" height=\"75px\">",
```

Comment:
Change image path to **https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/SlashNext%20SIEM/Playbook/logo/slashnext-logo.svg**